ha-trace/src/test/java/com/g2forge/habitat/trace/TestHTrace.java

```diff
@@ -70,6 +70,12 @@ public void caller() {
 		HAssert.assertEquals(getClass(), executable.getDeclaringClass());
 	}
 
+	@Test
+	public void entrypoint() {
+		final Executable entrypoint = HTrace.getEntrypoint(EntrypointFilter.ALL);
```

Comment:
This is the relevant line for this PR - the way to get the entrypoint into user code regardless of whether running under main() or any normal test frameworks.